Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -451,6 +445,14 @@ class BrowserViewController: UIViewController,
         }
     }
 
+    private func dismissModalsIfStartAtHome() {
+        let didStartAtHome = tabManager.startAtHomeCheck()
```

Comment:
nit: we don't need the `let` here. We can just check `if tabManager.startAtHomeCheck()` instead.

Reply:
Right, used it for better readability.